test/unit-minimal/main.tf

```diff
@@ -87,19 +95,11 @@ module "test" {
   source = "../.."
 
   # add only required arguments and no optional arguments
-  name     = "gke-unit-minimal"
-  location = "${var.region}-a"
-  node_locations = [
-    "${var.region}-b",
-  ]
-  network                = module.vpc.vpc.self_link
-  subnetwork             = module.subnetwork.subnetworks["${var.region}/${local.subnet.name}"].self_link
-  networking_mode        = "VPC_NATIVE"
-  master_ipv4_cidr_block = "10.4.96.0/28"
-  ip_allocation_policy = {
-    cluster_secondary_range_name  = local.subnet.secondary_ip_ranges.pods.name
-    services_secondary_range_name = local.subnet.secondary_ip_ranges.services.name
-  }
+  name = "gke-unit-minimal"
+
+  # deploy a public cluster with routes based networking mode
+  networking_mode      = "ROUTES"
```

Comment:
please deploy the recommended vpc native cluster

Reply:
we already do this in the unit-complete test

Reply:
what's the reasoning behind this?

Reply:
resolved